addons/hide-monitors/addon.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "Button to hide monitors",
+  "description": "Adds a button to projects that hides monitors, like variables or lists.",
+  "tags": ["editor", "projectPage"],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["projects"]
+    }
+  ],
+  "info": [
+    {
+      "id": "no-unintended",
+      "text": "You are not able to click through the places where the monitors would normally be. This is done so this addon can't be used to cheat in games, or using projects in other unintended ways."
```

Do you have an example of a project where you can cheat if the monitors are gone but not if they're just invisible?

---

Not really - would this wording be better?
```suggestion
      "text": "You are not able to click through the places where the monitors would normally be. This is done so you can't use the project in unexpected ways, like clicking on things you aren't supposed to."
```